instana/agent-google-market-app
# Migrate x-google-marketplace v1 to v2

* refactor code structure to match GCP examples
* change the apptester app, introduce tester image
* update helm chart to use the latest version
* disable serviceAccount and rbac in values.yaml
* create verify.sh script to build/push and verify changes
* update schema.yaml
  * change default endpoint
  * add tls configuration (only secretName)
  * change memory limits
